ui/lib/sync/addon/components/landing-cta.hbs

```diff
@@ -0,0 +1,39 @@
+<div class="box is-fullwidth is-sideless is-flex-between is-shadowless is-marginless">
+  {{#if (is-version "Enterprise")}}
+    <p>
+      Sync secrets to platforms and tools across your stack to get secrets when and where you need them.
+      <Hds::Link::Standalone
```

Comment:
Consulted design and updated link text from `Learn more` to `Secrets sync tutorial`